raster-foundry/raster-foundry
# Restore missing classes in batch assembly jar

## Overview

This PR ensures the batch assembly jar includes json codecs for query parameter classes.

### Checklist


### Notes

Why did they go away? I don't know. Maybe the code generator in macro paradise got tired. Maybe because everything was used only implicitly, assembly thought we didn't need it. Maybe our deprecated dropbox call caused assembly to get confused, like [this unfortunate soul](https://stackoverflow.com/a/37608677). Anyway, it's fixed. :heart: assembly

## Testing Instructions

 * assemble your batch jar on develop
 * run the commands in the test script in the linked issue from ammonite, or `scala -cp /abs/path/to/batch-assembly.jar` and just run the non-ammonite parts
 * you should get errors when you try to initialize the combined scene query params
 * reassemble your batch jar with this branch
 * run the same commands
 * you should be fine

Closes azavea/raster-foundry-platform#535
